packages/mediacenter/kodi/scripts/cputemp

```diff
@@ -24,51 +24,50 @@
 TEMP=0
 
 if [ $(basename "$0") = "gputemp" -o "$1" = "gpu" ]; then
-  if which lspci >/dev/null; then
+
+  if which lspci 2>/dev/null; then
     if lspci -n | grep 0300 | grep -q 10de; then
       [ -x /usr/bin/nvidia-smi ] && TEMP=`/usr/bin/nvidia-smi -q -x | grep '<gpu_temp>' | awk '{ print $1 }' | sed 's,<gpu_temp>,,g'`
```

Comment:
I guess we can also get rid of the backticks to be consistent with the rest of the script.

Might the following be a slightly more efficient implementation of this logic:
```
  if [ -x /usr/bin/nvidia-smi ]; then
    if command -v lspci >/dev/null && lspci -n | grep 0300 | grep -q 10de; then
      TEMP="$(/usr/bin/nvidia-smi -q -x | awk '/<gpu_temp>/ { print $1 }' | sed 's,<gpu_temp>,,g')"
    fi
  fi
```
It avoids checking for `lspci` when there's no `nvidia-smi` on the system. My thinking is that testing `-x` should be "cheaper" than executing another process such as `which`/`command` - `nvidia-smi` will only ever exist on Nvidia based systems, so a small effciciency gain for everyone else.